### PR TITLE
added RSS to telegram bot to rss

### PR DESCRIPTION
**Pull request by ippocratis**
I'm not sure if you want to include something like that
Its an open source telegram bot deployed with docker-compose
BUT it runs on a closed source messaging platform (great by any other means though)
So

### CODE REVIEW

**Comment by tborychowski** [comment]
Hi,
Sure, we can add that, no problem. If it's in docker-compose and can be self-hosted - great!
But if you could change 2 things, I'd be much happier :-)

1. Not sure if it's a typo, or intentional: should this be `rsstt` (rss-to-telegram), rather than `rrstt`?
2. Could you, please, remove everything that is not strictly essential to running it?
(That includes any comments from the docker-compose that are not absolutely required for starting the container)

I can see that they may have value and some people would find them useful, but my idea for this is that every app page gives the bare minimum, a simplified version of the setup, so that someone (myself included) can just copy & paste and run. I don't want to read anything that I don't need, before trying out the app. Only when I like it and want to set it up properly (for production) I can spend more time, dive deeper and go through the config options and see what else can be configured. 
What I'd really like to avoid is to have full-blown tutorials on all setup aspects that describe everything you may ever want to know but take time to read and understand. 
This is discouraging or even overwhelming for many of people, especially with cognitive challenges (that includes me).

I hope you understand :-)

**Comment by ippocratis** [comment]
That was a typo yes
it's rsstt (RSS to telegram)
And I agree with a bare minimum approach  in the docker-compose files.
Should I fix the file and open a new pull request?

**Comment by tborychowski** [comment]
Thanks for understanding.
You can correct it in this PR or create a new one, whichever is easier.

File: README.md [hunk 1]
```diff
@@ -361,6 +361,7 @@ The aims is to provide a ready-to-run recipes that you can just copy, paste and
  - [Miniflux](rss/../apps/rss/miniflux.md)
  - [Miniflux-filter](rss/../apps/rss/miniflux-filter.md)
  - [FreshRSS](https://www.freshrss.org/) 🔗 - second best :-)
+ - [RSS to telegram bot](apps/rss/rrstt.md)
```
Comment (tborychowski) [inline README.md]: missed one (rsstt) :-)

**Review by tborychowski — COMMENTED** [review]
looks much better, thank you!

File: README.md (hunk 1 above)
Comment (ippocratis) [reply README.md]: On God I forgot to change back the readme link 

I mean have some patience
I'm kinda new to github
:-)

Comment (tborychowski) [reply README.md]: no worries. 
I review code for living :-)

**Comment by ippocratis** [comment]
Added rsshub a nice rss tool I use to convert  social media groups/channels to rss feeds.

**Comment by tborychowski** [comment]
Cool, I wanted to test that one myself, just didn't have much time recently. Thanks.
So just 1 typo left to fix and we can merge it all :-)

**Review by tborychowski — APPROVED** [review]
thank you!

**Pull request merged**